pkg/securitycontext/util.go

```diff
@@ -134,6 +134,11 @@ func DetermineEffectiveSecurityContext(pod *v1.Pod, container *v1.Container) *v1
 		*effectiveSc.ReadOnlyRootFilesystem = *containerSc.ReadOnlyRootFilesystem
 	}
 
+	if containerSc.AllowPrivilegeEscalation != nil {
```

Comment:
:+1: on finding this, the `DetermineEffectiveSecurityContext` will cause subtle issues in PSP if not handled (and I've forgotten it a number of times)